package.json

```diff
@@ -5,6 +5,7 @@
     "node": ">=12.13.0"
   },
   "scripts": {
+    "audit": "yarn npm audit --recursive --environment production --json > yarn-audit-known-issues",
     "build": "yarn build:prod && yarn build:node",
     "build:docker": "docker build . -t rpx-xui-approve-org",
     "build:ng": "ng build",
```

Comment:
Would these really be needed? I think an know issue list update could be pushed to the remote branch straight without any build activity. If the build would fail it will not presumbly be because of the update in the file.